pkg/gotohelm/helmette/shims.go

```diff
@@ -70,3 +73,25 @@ func Unwrap[T any](from Values) T {
 	}
 	return out
 }
+
+// Sitobytes will translate string representation of Quantity to decimal value
+// See also: "_shims.sitobytes".
+func Sitobytes(value any) string {
+	v, ok := value.(string)
+	if !ok {
+		return ""
+	}
+	switch {
+	case strings.HasSuffix(v, "b") || strings.HasSuffix(v, "B"):
+		return v[:len(v)-1]
+	case strings.HasSuffix(v, "K"):
+		v = fmt.Sprintf("%sk", v[:len(v)-1])
+	case strings.HasSuffix(v, "g"):
+		v = fmt.Sprintf("%sG", v[:len(v)-1])
+	case strings.HasSuffix(v, "m"):
+		v = fmt.Sprintf("%sM", v[:len(v)-1])
+	}
```

Comment:
This is required to match `SI-to-bytes` capability, but I'm all for changing the Redpanda helm schema to match https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/